sdk/eventhub/azure-eventhub/azure/eventhub/_consumer.py

```diff
@@ -97,110 +93,86 @@ def __init__(self, client, source, **kwargs):
         self._owner_level = owner_level
         self._keep_alive = keep_alive
         self._auto_reconnect = auto_reconnect
-        self._retry_policy = errors.ErrorPolicy(
-            max_retries=self._client._config.max_retries,
-            on_error=_error_handler,  # pylint:disable=protected-access
-        )
+        self._retry_policy = self._amqp_transport.create_retry_policy(self._client._config)
         self._reconnect_backoff = 1
-        self._link_properties = {}  # type: Dict[types.AMQPType, types.AMQPType]
+        link_properties: Dict[uamqp_types.AMQPType, uamqp_types.AMQPType] = {}
```

Comment:
Why is this no longer self._link?

Reply:
I pass this into amqp_transport._create_link_properties, which will then create a new dict, which I set as self._link_properties further down. The reason I do this is b/c the keys in this link properties object are passed to uamqp.types.AMQPSymbol and the values are wrapped in another uamqp.types object. For pyamqp, it wraps keys and values in a similar symbol/long pyamqp type.